float64 set_point
---
bool valid_position